mesa/visualization/templates/js/pltviz.js

```diff
@@ -0,0 +1,24 @@
+var PyPlotModule = function(canvas_width, canvas_height) {
```

Comment:
I know the existing js code in Mesa uses `var`, but since this is new code, it is better to use `let`/`const` instead of `var`.